CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 * Verify container portal when checking dependency target membership.  
   [izaakschroeder](https://github.com/izaakschroeder)
   [#513](https://github.com/CocoaPods/Xcodeproj/issues/513)
+* [XCBuildConfiguration] Fix infinite recursion in #resolve_build_setting  
```

Comment:
nit extra line here

Reply:
It has two spaces after the last character. Does it need anything else?

Reply:
I meant from above, separate the two changelog entries

Reply:
also as a nit wrap the `#resolve_build_setting` with backticks

Reply:
Alright, got it!